openstack/identity/v3/projects/results.go

```diff
@@ -0,0 +1,48 @@
+package projects
+
+import (
+	"github.com/mitchellh/mapstructure"
```

Comment:
is this import already used by gophercloud, if not they might not be happy that you're forcing another dependency on them